src/Styles/MenuBar.less

```diff
@@ -3,17 +3,24 @@
     right: 25px;
     top: 20px;
     height: 30px;
+    border: 1px solid rgba(255,255,255,0.1);
```

Comment:
Sorry, they're not the same at all.  Something more like `rgba(60,60,60,0.8)` would be good I think.